serverless/experimental/gcp-log/README.md

```diff
@@ -1,14 +1,45 @@
-TODO(jayhou): make this info more friendly. Currently contains rough notes of
-how to set up a log on GCP.
+TODO(jayhou): Add more about what the log looks like and how to play with it after deploying.
 
 Note: this page is under construction.
 
-1.  Set environmental variable: GCP_PROJECT as your project name as a string.
+---
+
+# Serverless Log on GCP
+This directory contains another example implementation of a log represented by tiles and files, using components from the Trillian repo (e.g. compact ranges) and GCP infrastructure. It demonstrates the usage of [Google Cloud Functions](https://cloud.google.com/functions) to process and integrate new log entries and the usage of [Google Cloud Storage](https://cloud.google.com/storage) as the storage for tiles and files.
+
+## Overview
+We deploy two functions to GCF:
+1. `integrate`: with the `--initialise` flag, creates a bucket which acts as our log storage layer. Without it, integrates sequenced log entries to the tree by updating the tiles and checkpoint files.
+1. `sequence`: assigns leaf index numbers to each new log entry, preparing it for integration to the tree.
+
+Both functions are HTTP-triggered and run when their respective endpoints are requested.
+
+## Deployment
+### Pre-reqs:
+1.  Create a GCP project. [Enable](https://cloud.google.com/endpoints/docs/openapi/enable-api) GCF and GCS APIs.
+
+### GCF function deployment:
 1.  Generate a set of public and private keys following
     [these](https://github.com/google/trillian-examples/tree/master/serverless#generating-keys)
     instructions and set them as the `SERVERLESS_LOG_PUBLIC_KEY` and
     `SERVERLESS_LOG_PRIVATE_KEY`.
-1.  Run `gcloud functions deploy integrate --entry-point Integrate --runtime
-    go116 --trigger-http` to deploy the Integrate Google Cloud Function.
-1.  Run `gcloud functions deploy sequence --entry-point Sequence --runtime go116
-    --trigger-http --max-instances 1` to deploy the Sequence Function.
+1. Set `PROJECT_NAME` as the name of your GCP project (string, not number).
+1.  Deploy the Integrate function:
+    ```
+    gcloud functions deploy integrate \
+    --entry-point Integrate \
+    --runtime go116 \
+    --trigger-http \
+    --set-env-vars GCP_PROJECT=${PROJECT_NAME} SERVERLESS_LOG_PUBLIC_KEY=${PUBLIC_KEY} SERVERLESS_LOG_PRIVATE_KEY=${PRIVATE_KEY} \
+    --source=./serverless/experimental/gcp-log 
```

Comment:
Would it be a good idea to add `--max-instances=1` here too?

Reply:
Done.